zerver/data_import/slack.py

```diff
@@ -823,10 +824,14 @@ def get_messages_iterator(
                     # skipping those messages is simpler.
                     continue
                 if message.get("mimetype") == "application/vnd.slack-docs":
-                    # This is a Slack "Post" which is HTML-formatted,
-                    # and we don't have a clean way to import at the
-                    # moment.  We skip them on import.
-                    continue
+                    # This is a Slack "Post" which is HTML-formatted
+                    file_url = message.get("url_private_download")
```

Comment:
Does `message` always have "url_private_download"? This might be `None`.

Reply:
As mentioned in  [standard export format for Slack canvas](https://slack.com/intl/en-in/help/articles/220556107-How-to-read-Slack-data-exports#how-to-view-canvases) we always get `"url_private_download"` and `"mimetype" : "application/vnd.slack-docs"`